hack/.ci/run-e2e-gke-release.sh

```diff
@@ -20,8 +20,11 @@ trap gather-artifacts-on-exit EXIT
 SO_NODECONFIG_PATH="${SO_NODECONFIG_PATH=${parent_dir}/manifests/cluster/nodeconfig.yaml}"
 export SO_NODECONFIG_PATH
 
-# Make sure there is no default storage class before we create our own so we always use our own provisioner from the release.
-unset-default-storageclass
+SO_SCYLLACLUSTER_STORAGECLASS_NAME="${SO_SCYLLACLUSTER_STORAGECLASS_NAME=}"
+export SO_SCYLLACLUSTER_STORAGECLASS_NAME
+if [[ -z "${SO_CSI_DRIVER_PATH+x}" || -n "${SO_CSI_DRIVER_PATH}" ]]; then
+  SO_SCYLLACLUSTER_STORAGECLASS_NAME="${SO_SCYLLACLUSTER_STORAGECLASS_NAME:-scylladb-local-xfs}"
```

Comment:
won't this variable be always set given the defaulting above? structuring this as if+else might be less bug prone

Reply:
no longer applicable - I applied changes we discussed internally